__tests__/app.test.js

```diff
@@ -272,4 +322,36 @@ test ('returns a 400 if inc_votes is missing.', () => {
         });
 });
 
+
+test('returns a 404 found if the username is not found.', () => {
```

Comment:
Should these tests be in this describe block? It looks like you have a test for PATCH requests above this, so you may need to rejig some of these around.